modules/bibsword/lib/bibsword_client_dblayer.py

```diff
@@ -15,349 +17,390 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-'''
-BibSWORD Client DBLayer
-'''
+"""
+BibSword Client DBLayer.
+"""
 
-import datetime
 import time
+from invenio.dateutils import convert_datestruct_to_datetext
 from invenio.dbquery import run_sql
-from invenio.bibsword_config import CFG_SUBMISSION_STATUS_PUBLISHED, \
-                                                CFG_SUBMISSION_STATUS_REMOVED
 
 
-def get_remote_server_auth(id_remoteserver):
-    '''
-        This function select the username and the password stored in the
-        table swrREMOTESERVER to execute HTTP Request
-        @param id_remoteserver: id of the remote server to contact
-        @return: (authentication_info) dictionnary conating username - password
-    '''
-
-    authentication_info = {'error':'',
-                           'hostname':'',
-                           'username':'',
-                           'password':'',
-                           'realm':'',
-                           'url_servicedocument':''}
-
-    qstr = '''SELECT host, username, password, realm, url_servicedocument ''' \
-           ''' FROM swrREMOTESERVER WHERE id=%s'''
-    qres = run_sql(qstr, (id_remoteserver, ))
-
-    if len(qres) == 0 :
-        authentication_info['error'] = '''The server id doesn't correspond ''' \
-                                       '''to any remote server'''
-        return authentication_info
+# CREATE TABLE `swrCLIENTTEMPSUBMISSION` (
```

Comment:
It might better if we add this to https://github.com/CERNDocumentServer/invenio/blob/prod/modules/miscutil/sql/tabcreate.sql (not mentioning that it might be nice to have upgrade recipes, but I guess we are good without them)